os/arch/arm/include/bcm4390x/chip.h

```diff
@@ -0,0 +1,87 @@
+/****************************************************************************
+ *
+ * Copyright 2018, Cypress Semiconductor Corporation or a subsidiary of
+ * Cypress Semiconductor Corporation. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __ARCH_ARM_INCLUDE_BCM4390X_CHIP_H
+#define __ARCH_ARM_INCLUDE_BCM4390X_CHIP_H
+
+/*****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#ifndef __ASSEMBLY__
+
+#include <stdint.h>
+
+#endif
+/*****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+#if defined(CONFIG_AUDIO) && defined(PLATFORM_AUDIO_NUM_OUTPUTS)
```

Comment:
Where is a definition of PLATFORM_AUDIO_NUM_OUTPUTS?

Reply:
it will be removed as your guidance.
BR, HJ Lee.